src/main/resources/run-class.sh

```diff
@@ -98,9 +98,17 @@ if [ -e conf/env.conf ]; then
   CONF_FILE="-Dconfig.file=conf/env.conf"
 fi
 
+#Add a cache-key config, user tests for existence
+ADD_CACHE_KEY=""
+if [ -e conf/cacheKey.conf ]; then
+  CACHEKEY_FILE_CONTENTS = $(<cacheKey.conf)
+  CACHEKEY=${CACHEKEY_FILE_CONTENTS##*: }
+  ADD_CACHEKEY="-Dapplication.cacheKey=$CACHEKEY"
+fi
+
 CLASSPATH=`find lib -name '*.jar' | tr "\\n" :`
 JAVA_CMD=(java $JVM_ARGS -classpath $CLASSPATH $CONF_FILE
-  ${LOGBACK_CONF[@]})
+  ${LOGBACK_CONF[@]} $ADDCACHEKEY)
```

Comment:
You should put your variable in double-quotes (although you'd also need to fix your regex to drop the quotes in your conf file).